packages/web-react/src/components/Dropdown/README.md

```diff
@@ -15,6 +15,22 @@ import { Dropdown, Button } from '@lmc-eu/spirit-web-react/components';
 </Dropdown>
 ```
 
+### Dropdown with Item
+
+Enhance your DropdownPopover by incorporating the versatile [Item] component.
+Explore additional examples and insights within the dedicated documentation for the [Item] component.
+
+```jsx
+import { UncontrolledDropdown, DropdownTrigger, DropdownPopover, Item } from '@lmc-eu/spirit-web-react/components';
+
+<DropdownModern id="DropdownExample" isOpen={isOpen} onToggle={onToggle}>
```

Comment:
Also, you are using `isOpen` and `onToggle` which are not set. What about importing `useToggle` or some similar hook we have.

Reply:
fixed - added constants as in other examples